odml/base.py

```diff
@@ -176,10 +180,16 @@ def insert(self, position, section):
         self._sections.append(section)
         section._parent = self
 
-    def append(self, section):
+    def append(self, *vsection):
         """adds the section to the section-list and makes this document the section’s parent"""
-        self._sections.append(section)
-        section._parent = self
+        ll = len(vsection)
+        for i in list(range(0,ll)):
+	     # from odml.section import BaseSection
+             # This check gives ERRORS in validator tests
+             #if isinstance(vsection[i], BaseSection):
+             #	raise KeyError("Object " + str(vsection[i]) + " is not a Section.")
+             self._sections.append(vsection[i])
+             vsection[i]._parent = self
 
```

Comment:
@jgrewe Thank you for the detailed review. Yes, I should decrease the number of commits :) It is an old habit and due to I build the branch over git at each new test. 

Regarding validator test errors in detecting attempted addition of a property to the main Document object, this is also the issue #69. I will get back on this.